resource-usage-collect-addon/deploy/resources/managed-cluster-set-binding.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  generation: 1
+  name: global
+  namespace: default
+spec:
+  clusterSet: global
+
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
```

Comment:
suggest to move placement to a seperate yaml.

Reply:
got it